File: sdk/python/src/dagger/telemetry/attributes.py
from typing import Final

UI_MASK: Final = "dagger.io/ui.mask"
"""Replace parent span (e.g., `exec /runtime`)."""

UI_PASSTHROUGH: Final = "dagger.io/ui.passthrough"
"""Reveal only child spans (e.g., `python runtime execution` parent span)."""

UI_ENCAPSULATE: Final = "dagger.io/ui.encapsulate"
"""Hide children by default (e.g., test case that runs pipelines)."""

File: sdk/typescript/telemetry/attributes.ts
/**
 * Replace parent span (e.g., `exec /runtime`).
 */
export const UI_MASK = "dagger.io/ui.mask"

/**
 * Reveal only child spans (e.g., `typescript runtime execution` parent span).
 */
export const UI_PASSTHROUGH = "dagger.io/ui.passthrough"

/**
 * Hide children by default (e.g., test case that runs pipelines).
 */
export const UI_ENCAPSULATE = "dagger.io/ui.encapsulate"
